protocol/v2/qbft/instance/prepare.go

```diff
@@ -100,7 +100,7 @@ func getRoundChangeJustification(state *specqbft.State, prepareMsgContainer *spe
 	return ret, nil
 }
 
-// validSignedPrepareForHeightRoundAndRoot known in dafny spec as validSignedPrepareForHeightRoundAndDigest
+// validSignedPrepareForHeightRoundAndRootIgnoreSignature known in dafny spec as validSignedPrepareForHeightRoundAndDigest
```

Comment:
@kchojn What do you think about changing the function name instead (`validSignedPrepareForHeightRoundAndRoot`)? I checked the spec and `validSignedPrepareForHeightRoundAndDigest` doesn't verify the signature, so IMO the `IgnoreSignature` suffix is kind of redundant

```dfy
    predicate validSignedPrepareForHeightRoundAndDigest(
        sPayload: SignedPrepare, 
        height:nat, 
        round:nat, 
        digest:Hash,
        validators: seq<Address>
        )
    {
        var uPayload := sPayload.unsignedPayload;
        && uPayload.height == height
        && uPayload.round == round
        && uPayload.digest == digest
        && recoverSignedPrepareAuthor(sPayload) in validators
    }
```